qutebrowser/browser/webengine/webengineinspector.py

```diff
@@ -50,7 +50,8 @@ def createWindow(self,
 
         See WebEngineView.createWindow for details.
         """
-        return self.page().inspectedPage().view().createWindow(wintype)
+        newpage = self.page().inspectedPage().createWindow(wintype)
+        return self.forPage(newpage)
```

Comment:
The `self` here is rather confusing: It suggests that it would create a `WebEngineInspectorView` with the given `WebEnginePage` instead of using the correct qutebrowser type, `WebEngineView` (both in `qutebrowser/browser/webengine/webview.py`).

It took me a good while of debugging until I understood why this works at all: [`QWebEngineView::forPage()` is a static Qt method](https://doc.qt.io/qt-6/qwebengineview.html#forPage), i.e. gets turned into a Python classmethod. It also returns the existing view, rather than creating a new one, like the name suggests...

To avoid confusion, I'd suggest calling the classmethod on the class (as usually done for classmethods), but also using the type which this actually is:

```suggestion
        return webview.WebEngineView.forPage(newpage)
```

(will also need to add `, webview` to `from qutebrowser.browser.webengine import webenginesettings`)